Gemfile

```
@@ -18,7 +18,9 @@ gem 'uglifier', '>= 1.3.0'
 
 gem 'm'
 
-gem 'danger'
+gem 'danger', git:'https://github.com/colorbox/danger.git', branch: 'add-pp'
```

```suggestion
gem 'danger', git: 'https://github.com/colorbox/danger.git', branch: 'add-pp'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>